open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-A logical first question might be:
-
-1. Do you like parties?
-
-If they don't like parties at all, they can probably skip this one. But just in case, a good follow-up might be:
-
-* Is there anyone there who you want to meet?
-
-Followed by
-
-* Do you have any big assigments or studying to do?
-
-If they do like parties, we might ask:
-
-2. Do you have a big assignment due?
-
-## Map it out
-
-For a small problem like this, it might be helpful to write out a small flow chart or map of the logic. For a bigger problem, you might come up with a different way to organize it.
-
-Below is a flowchart that maps out part of the logic we worked out above:
-
-[![](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggVERcblx0QVtEbyB5b3UgbGlrZSBwYXJ0aWVzP10gLS0-IEJ7eWVzfVxuICBBIC0tPiBDe25vfVxuICBDIC0tPkRbSXMgdGhlcmUgYW55b25lIHlvdSB3YW50IHRvIG1lZXQ_XVxuICBEIC0tPkV7eWVzfVxuICBEIC0tPkZ7bm99XG4gIEYgLS0-IFooZmE6ZmEtYmVkIERvbid0IEdvKVxuICBFIC0tPiBHW0RvIHlvdSBoYXZlIGEgYmlnIGFzc2lnbm1lbnQgZHVlP11cbiAgRyAtLT5oe3llc31cbiAgRyAtLT5pe25vfVxuICBpIC0tPlkoZmE6ZmEtZ2lmdCBHbyB0byB0aGUgcGFydHkpXG5cbiAgaCAtLT4gWlxuICBCIC0tPiBHXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcblx0QVtEbyB5b3UgbGlrZSBwYXJ0aWVzP10gLS0-IEJ7eWVzfVxuICBBIC0tPiBDe25vfVxuICBDIC0tPkRbSXMgdGhlcmUgYW55b25lIHlvdSB3YW50IHRvIG1lZXQ_XVxuICBEIC0tPkV7eWVzfVxuICBEIC0tPkZ7bm99XG4gIEYgLS0-IFooZmE6ZmEtYmVkIERvbid0IEdvKVxuICBFIC0tPiBHW0RvIHlvdSBoYXZlIGEgYmlnIGFzc2lnbm1lbnQgZHVlP11cbiAgRyAtLT5oe3llc31cbiAgRyAtLT5pe25vfVxuICBpIC0tPlkoZmE6ZmEtZ2lmdCBHbyB0byB0aGUgcGFydHkpXG5cbiAgaCAtLT4gWlxuICBCIC0tPiBHXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
-
-## Building our Docassemble interview
 
-### What kinds of information do we need to gather?
-
-Before we start, let's think about what kinds of information we will want to ask.
-
-This Docassemble interview will use two kinds of screens:
-
-1. Yes/no questions
-2. Ending screens
-
-As a reminder, ending screens use the `event` specifier. 
-
-We haven't seen a `question` block that uses just a yes/no question yet. Here's a [link to the documentation](https://docassemble.org/docs/questions.html#tocAnchor-1-6-1).
-
-Let's take a look at an example:
+We'll need two ending screens. You make endings screens with an [`event` attribute](https://docassemble.org/docs/questions.html#event) in a `question` block. That's like the `field` attribute we've used before. Unlike regular question screens, your user isn't going to give an answer here, so there's no variable for Docassemble to find. The `event` attribute tells Docassemble how to find the question without needing a variable name.
 
 ```yaml
+---
+id: no_go_advice
+event: dont_go_to_party
+question: |
+  Don't go to that dumb party. Do something fun instead!
+---
+id: go_advice
+event: go_to_party
 question: |
-  Are you at least 18 years of age?
-yesno: over_eighteen
```

Comment:
Note: Since we already have a story going on with a party, this seems like a great place to keep that narrative going.